src/types/events/base-events.ts

```diff
@@ -749,8 +785,10 @@ export interface SharedChannelInviteAccepted {
   accepting_user: SharedChannelUserItem;
   event_ts: string;
 }
+// for backward-compatibility
```

Comment:
Thank you for doing this!

For my own future hints, esp since I would like to release bolt v4 this year, I add a "TODO (breaking change)" comment in areas for deprecation. Also might be useful to add a JSdoc to signify deprecation and what to move to; this would be raised up to the developer in their IDE at author-time.

```suggestion
// TODO: (breaking change) for backward-compatibility; remove non-Event-suffix type in next major version.
/**
 * @deprecated Will be removed in next major version. Use the `SharedChannelInviteApprovedEvent` interface instead.
 */
```

Could we add a similar comment next to each inconsistently-named backwards-compatible type in this PR?

Reply:
Thanks! I will update the comments tomorrow in my timezone!

Reply:
@filmaj Updated the comments. Can you check again and mege this PR if it's fine tomorrow?